lib/config/presets/internal/regex-managers.ts

```diff
@@ -78,7 +78,7 @@ export const presets: Record<string, Preset> = {
         customType: 'regex',
         fileMatch: ['.+\\.tfvars$'],
         matchStrings: [
-          '#\\s*renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\s.*?_version\\s*=\\s*"(?<currentValue>.*)"',
+          '#\\s*renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?(?: extractVersion=(?<extractVersion>.*?))?\\s.*?_version\\s*=\\s*"(?<currentValue>.*)"',
```

Comment:
We do you have here a different char selector than in the other presets?

Reply:
It's consistent wihtin itself. Would you like to make this entire regex consistent with the others?

Reply:
I see. Keep it that way in that case.